thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -64,4 +64,6 @@ struct MediaAtom {
   10: optional string description
   11: optional Metadata metadata
   12: optional shared.Image posterImage
+  13: optional string videoSource
```

Comment:
`videoSource` implies the media atom is only for video. I (hope) audio would also be included in this atom (albeit not soon!).

I can't remember why the `source` field exists above, however given no atoms currently populate this field, should we repurpose it?